test/e2e/autoscale_test.go

```diff
@@ -232,7 +209,8 @@ func TestAutoscaleUpDownUp(t *testing.T) {
 		clients.KubeClient,
 		deploymentName,
 		isDeploymentScaledUp(),
-		"DeploymentIsScaledUp")
+		"DeploymentIsScaledUp",
+		2*time.Minute)
```

Comment:
This is reducing the time we wait for initial scaleup from the old timeout of 6 minutes down to 2 minutes. Do we know from previous test runs that this always happens within 2 minutes, so that we're not introducing another potential flake?

Reply:
I'm pretty certain that if scale up doesn't happen within 2 minutes, it never will.  I saw this while developing these changes.

Reply:
I poked through several previous test runs and the scaling up tens to happen in less than one second. I didn't see any cases where it even approached 2 minutes, so this seems fine.

Reply:
Yeah, what makes it okay is that we aren't starting the time until we have 200 responses from all the requests we sent.  So processing time can't affect this.  It's just purely metrics pipeline and autoscaler latency.